### DISCUSSION TITLE
Add K-Medoids Clustering Support

### DISCUSSION THREAD

**Discussion by anjali-rgpt** [nidhaloff/igel#85] [Ideas]
I would actually like to add the K-Medoids clustering algorithm to igel, as it is an often used alternative to K-Means. And it has less of an error penalty. Would it be okay to create an issue / pull request for the same?

**Comment by nidhaloff** [comment]
Hi @anjali-rgpt Thanks for your suggestion. Sure, feel free to create an issue and a PR ;)
